nautobot/circuits/templates/circuits/circuittermination_create.html

```diff
@@ -25,20 +25,28 @@
                     <p class="form-control-static">{{ form.term_side.value }}</p>
                 </div>
             </div>
+                {% with location_tab_active=form.initial.location %}
                 {% with providernetwork_tab_active=form.initial.provider_network %}
+                {% with cloudnetwork_tab_active=form.initial.cloud_network %}
                     <ul class="nav nav-tabs" role="tablist">
-                        <li role="presentation"{% if not providernetwork_tab_active %} class="active"{% endif %}><a href="#location" role="tab" data-toggle="tab">Location</a></li>
+                        <li role="presentation"{% if location_tab_active or not providernetwork_tab_active and not cloudnetwork_tab_active %} class="active"{% endif %}><a href="#location" role="tab" data-toggle="tab">Location</a></li>
```

Comment:
It will default to the location tab during `CircuitTerminationCreateView` and it will default to the tab of the field that is not null in `CircuitTerminationEditView`